pilot/docker/Dockerfile.proxy_debug

```diff
@@ -26,5 +26,8 @@ RUN useradd -m --uid 1337 istio-proxy && \
     echo "istio-proxy ALL=NOPASSWD: ALL" >> /etc/sudoers && \
     chown -R istio-proxy /var/lib/istio
 
+# For coredumps.
+RUN mkdir -p /var/istio/proxy
```

Comment:
We use /var/lib/istio for pretty much everything - please, not another filesystem layout !

Reply:
We also use /var/log/istio for log on mesh expansion case - I would say that's the right place.